homeassistant/components/seventeentrack/repairs.py

```diff
@@ -0,0 +1,76 @@
+"""Repairs for the SeventeenTrack integration."""
+
+from typing import cast
+
+import voluptuous as vol
+
+from homeassistant.components.repairs import ConfirmRepairFlow, RepairsFlow
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import issue_registry as ir
+
+from . import DOMAIN
+from .const import DEPRECATED_KEY
+
+
+@callback
+def deprecate_sensor_issue(hass: HomeAssistant, entry_id: str) -> None:
+    """Ensure an issue is registered."""
+    ir.async_create_issue(
+        hass,
+        DOMAIN,
+        "deprecate_sensor",
```

Comment:
Let's also put the entry_id in the issue_id, so if someone set this up twice, they get 2 issues, otherwise I can see them messing up which entry they are repairng

Reply:
good idea.
added